scripts/sv/manage_sv_pipeline.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# This script manages the SV discovery pipeline by calling other scripts
+# a) it creates a Google Dataproc cluster used for running the GATK-SV
+#    pipeline (call create_cluster.sh)
+# b) it runs the analysis (call runWholePipeline.sh)
+# c) it copies results to a datetime and git-version stamped directory
+#    on GCS, along with logged console output (call copy_sv_results.sh)
+# d) it shuts down the cluster
+
+# terminate script on error or if a command fails before piping to another command
+set -eu
+set -o pipefail
+
+if [[ "$#" -lt 4 ]]; then
+    echo -e \
+"Please provide:
+  [1] local directory of GATK build (required)
+  [2] project name (required)
+  [3] GCS path to indexed BAM (required)
+  [4] GCS path to reference fasta (required)
+      OPTIONAL arguments:
+  [5] path to initialization script (local or GCS, defaults to
+      ${GATK_DIR}/scripts/sv/default_init.sh if omitted or empty)
```

Comment:
I have a concern here: if the 5th and 6th argument are omitted, is the `shift 6` statement below going to skip two arguments that are intended to be sent to java be "shifted" away?